python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -177,10 +216,12 @@ def get_op_usage(self, op: PhysicalOperator) -> ExecutionResources:
     def get_op_usage_str(self, op: PhysicalOperator) -> str:
         """Return a human-readable string representation of the resource usage of
         the given operator."""
-        usage_str = f"cpu: {self._op_usages[op].cpu:.1f}"
-        if self._op_usages[op].gpu:
-            usage_str += f", gpu: {self._op_usages[op].gpu:.1f}"
-        usage_str += f", objects: {self._op_usages[op].object_store_memory_str()}"
+        usage_str = f"cpu: {self._op_running_usages[op].cpu:.1f}"
+        if self._op_running_usages[op].gpu:
+            usage_str += f", gpu: {self._op_running_usages[op].gpu:.1f}"
+        usage_str += (
+            f", objects: {self._op_running_usages[op].object_store_memory_str()}"
+        )
```

Comment:
also include pending usage?

Reply:
not a blocker for this PR. maybe we should unify the resource formatting for both the global and the per-op progress bars.

Reply:
Agree! Tracking with #47190.